openvdb_houdini/houdini/GEO_PrimVDB.h

```diff
@@ -47,7 +47,13 @@
  */
 
 #include <UT/UT_Version.h>
-#if !defined(SESI_OPENVDB) && (UT_VERSION_INT >= 0x0c050157) // 12.5.343 or later
+
+// Using the native OpenVDB Primitive shipped with Houdini is strongly recommended,
+// as there is no guarantee that this code will be kept in sync with Houdini.
+// However, for debugging it can be useful, so supply -DSESI_OPENVDB_PRIM to
```

Comment:
Should that be added as an option to the CMake script?

Reply:
I'm thinking that we don't as we're not considering maintaining it through CI. The extra requirement of having to actually provide it to the compiler yourself (or otherwise modify the build system or source) seems like enough of a barrier that no-one will switch it on by accident. What do you think?

Reply:
That's fine.  It's true that making it easy to enable could lead to confusion.